languages/kotlin/concepts/basics/about.md

```diff
@@ -1 +1,41 @@
-TODO: add information on basics concept
+# What's next?
+
+## Type inference
+
+Kotlin compiler can infer types for functions and variables in most of the cases. However, declaring function return types explicitly for **public API** is a good practice.
+
+## Entry point
+
+To run your Kotlin program you need to define so-called entry point: function with name `main` with or without arguments. However, in Exercism course we are using automatic tests that are using other functions defined in exercises.
```

Comment:
Do you think it's worth mentionings, that the `main` must be defined in a script, not in a class, as in Java?

Reply:
Yeah that seems wise! Could you suggest a change?

Reply:
_... [top-level function](https://kotlinlang.org/docs/reference/functions.html#function-scope) with name `main` ..._

Fits?

Reply:
Could you perhaps use Github's code suggestion feature? See https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request

This makes it very easy for me to commit changes :)

Reply:
Do not have this interface for unknown reason.

Reply:
@eparovyshnaya suggestion syntax is described here (pt. 6):
https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request

It's a bit tricky to discover it indeed :)